USERS.md

```diff
@@ -77,3 +77,4 @@ Currently, the following organizations are **officially** using Argo Workflows:
 1. [Tulip](https://tulip.com/)
 1. [Wavefront](https://www.wavefront.com/)
 1. [Wellcome Trust](https://wellcome.ac.uk/)
+1. [New Relic](https://newrelic.com/)
```

Comment:
can I ask you to put this in alphabetical order please?

Reply:
Of course! I did not notice that it was in an order, oops.